Source/controls/input.h

```diff
@@ -9,7 +13,11 @@ namespace devilution {
 
 inline int PollEvent(SDL_Event *event)
 {
-	int result = SDL_PollEvent(event);
+#ifdef USE_SDL3
+	const int result = SDL_PollEvent(event) ? 0 : -1;
+#else
+	const int result = SDL_PollEvent(event);
+#endif
 	if (result != 0) {

```

Comment:
I much prefer the new bool return, i realize we will have to adjust a few other places for this.
```suggestion
#ifdef USE_SDL3
	const bool success = SDL_PollEvent(event);
#else
	const bool success = SDL_PollEvent(event) >= 0;
#endif
	if (success) {
```